Source/ThirdParty/o3dgc.js

```diff
@@ -181,16 +181,27 @@ define(['../Core/getTimeStamp'], function(getTimeStamp) {
         }
         return pos;
     }
+
     // Timer class
+    var windowPerformance = (typeof performance !== 'undefined') ? performance : {};
+    if (!windowPerformance.now) {
+        local.nowOffset = Date.now();
+        if (windowPerformance.timing && windowPerformance.timing.navigationStart) {
+            local.nowOffset = windowPerformance.timing.navigationStart;
+        }
+        windowPerformance.now = function now() {
```

Comment:
Looks that way, yeah.  Instead of creating the windowPerformance local var, just create and use a `now` local var instead.